app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -115,7 +115,7 @@ Vue.component('grantsCartEthereumPolygon', {
 
       if (appCart.$refs.cart.network === 'mainnet') {
         appCart.$refs.cart.networkId = '137';
-        url = 'https://rpc-mainnet.maticvigil.com';
+        url = 'https://polygon-rpc.com';
```

Comment:
@chibie - this is the recommend default RPC - when I use this I don't see the `429` errors that I was seeing with the other. Is there any reason we should stick with maticvigil?